documentation/asciidoc/computers/using_linux/linux-crontab.adoc

```diff
@@ -52,6 +52,15 @@ View your currently saved scheduled tasks with:
 crontab -l
 ----
 
+=== Erase scheduled tasks
+
+Delete all currently scheduled tasks:
```

Comment:
If you have multiple cron tasks set up, then I guess "deleting all of them" is a bit of a heavy-handed approach? Is it easier to just selectively remove cron jobs by editing the crontab? :shrug:

Reply:
Right, It's easier to edit the Crontab one by one. Thank you for your sharp point.